.github/workflows/ci.yml

```diff
@@ -35,10 +35,10 @@ jobs:
       id: buildx
       uses: docker/setup-buildx-action@v2
       with:
-        driver: docker
+        driver-opts: network=host
     - name: Build
       shell: bash
-      run: make
+      run: make DOCKER_ORG="ghcr.io/bufbuild" DOCKER_BUILD_EXTRA_ARGS="--build-arg BUILDKIT_INLINE_CACHE=1 --push"
```

Comment:
`BUILDKIT_INLINE_CACHE=1` will publish cache information along with images to the container registry.